vortex-duckdb/src/duckdb/scalar_function.rs

```diff
@@ -14,7 +14,8 @@ impl ScalarFunction {
             let name_ptr = cpp::duckdb_vx_sfunc_name(self.as_ptr());
             std::ffi::CStr::from_ptr(name_ptr)
                 .to_str()
-                .vortex_expect("invalid utf-8")
+                .map_err(|_| vortex_err!("invalid utf-8"))
```

Comment:
you now swallow the Utf8Error though, which might have given useful info (e.g., which charpoint in a long string)

Reply:
we can forward the error